<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SchemaMenu</class>
 <widget class="QGroupBox" name="SchemaMenu">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>schema</string>
  </property>
  <property name="title">
   <string>No schema</string>
  </property>
  <layout class="QVBoxLayout" name="layoutSchema">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>1</number>
     </property>
     <widget class="QWidget" name="tabTables">
      <attribute name="title">
       <string>Tables</string>
      </attribute>
      <layout class="QHBoxLayout" name="layoutTables">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QListWidget" name="listTables">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="groupTableButtons" native="true">
         <layout class="QGridLayout" name="layoutTableButtons">
          <item row="3" column="1">
           <widget class="QToolButton" name="buttonDeleteAllTables">
            <property name="text">
             <string>Delete All</string>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QToolButton" name="buttonRefreshTables">
            <property name="text">
             <string>Refresh</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QToolButton" name="buttonDeleteTable">
            <property name="text">
             <string>Delete</string>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QToolButton" name="buttonCreateAllTables">
            <property name="toolTip">
             <string>Create all MCHS Media tables</string>
            </property>
            <property name="text">
             <string>Create All</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QToolButton" name="buttonExportTable">
            <property name="text">
             <string>Export</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QToolButton" name="buttonImportTable">
            <property name="text">
             <string>Import</string>
            </property>
           </widget>
          </item>
          <item row="4" column="0">
           <spacer name="spacerTableButtons">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>230</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabUpdate">
      <attribute name="title">
       <string>Update</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QGroupBox" name="groupUpdateRange">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="title">
          <string>Update range</string>
         </property>
         <layout class="QHBoxLayout" name="layoutUpdateRange">
          <item>
           <widget class="QCheckBox" name="checkUpdateToLast">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="toolTip">
             <string>Update up to latest news date in database.</string>
            </property>
            <property name="text">
             <string>Last recorded</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QDateTimeEdit" name="valueUpdateDateA">
            <property name="toolTip">
             <string>Lower time edge.</string>
            </property>
            <property name="dateTime">
             <datetime>
              <hour>0</hour>
              <minute>0</minute>
              <second>0</second>
              <year>1752</year>
              <month>9</month>
              <day>14</day>
             </datetime>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QDateTimeEdit" name="valueUpdateDateB">
            <property name="toolTip">
             <string>Upper time edge.</string>
            </property>
            <property name="date">
             <date>
              <year>9999</year>
              <month>12</month>
              <day>31</day>
             </date>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="buttonSetDateNow">
            <property name="text">
             <string>Now</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="checkUpdateFromAny">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="toolTip">
             <string>Retreive all recent news.</string>
            </property>
            <property name="text">
             <string>Any</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="layoutUpdateButtons">
         <item>
          <widget class="QToolButton" name="buttonUpdate">
           <property name="text">
            <string>Update now</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="buttonSchedule">
           <property name="text">
            <string>Schedule</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
